README.md

```diff
@@ -1,48 +1,74 @@
+
+  After moving the ffmpeg folder into your C: drive, open Command Prompt with administrator and run this: setx /m PATH "C:\ffmpeg\bin;%PATH%"
+
+  If you get a success message, type in "ffmpeg" to the command prompt. If you get an error, you need to restart your computer, once finished restarting, run ffmpeg inside of the command prompt window once more, and if you do not get an error you're all set.
+
+# How to Run
+
+  Once you've downloaded the WackyWebM folder by going to the GitHub repository and selecting the green "Code" button, then clicking "Download ZIP"
+
+  Once the ZIP is fully installed, you need to extract the folder inside. After doing this, you are set. 
+  
+  Use `node wackywebm.js [mode] <file>` to use WackyWebM without running the run.bat.
+
+If you're on windows, you can simply double-click run.bat, as long as it's in the same folder as wackywebm.js.
+
+If you're on MacOS or Linux, you can run run.sh. You may have to give it permission to execute with chmod +x run.sh.
+
+You can also tweak the script to taste if you want to adjust how your file is generated.
+
+You can also change the bitrate of the output file by tweaking -b (Default is 1M, which means 1 MB/s. If the file is too large, consider lowering this value.)
+
+# Modes
+`Bounce` (Default): The video bounces up and down.
+
+`Shutter`: The video bounces left to right.
+
+`Sporadic`: The video glitches and wobbles randomly.
+
+`Shrink`: The video shrinks vertically until it's just one pixel thin.
+
+`AudioBounce`: The video's vertical height changes relative to the current audio level verses the highest within the video.
+
+`AudioShutter`: The video's horizontal width changes relative to the current audio level verses the highest within the video.
+
+`Jumpscare`: The video shrinks down small, then gets big at the specified frame.
+
+`Keyframes`:  The video's height and width change based on a number of keyframes outlined in the file given as an argument. The format is as follows:
+Every line consists of 4 comma-seperated values:
+first, the time in the video of the keyframe; either one integer representing seconds, or two, seperated by any one of the characters ., : or -, where the first still represents seconds, and the second represents frames.
+next, the width, then the height at that keyframe (in pixels)
+finally, the interpolation with which to advance towards the next keyframe (currently, only linear is supported.)
+If it isn't overwritten, an implicit keyframe at 0 frames into the video is added with linear interpolation and the video's original size.
+To use this mode, add -k with the path to your csv file.
+
+  Additionally, any 2 modes can be combined using a + symbol, like Bounce+Shutter. If one only specifies width, and one only specifies height, then those respective values are used. If there is a conflict, the value from the latter of the 2 modes is used (so Shrink+Bounce would result in the same effect as just Bounce)
+
+
+# Support

```

Comment:
Could you make this a subheading (`##` instead of `#`)

Reply:
done